code/modules/reagents/chemistry/machinery/pandemic.dm

```diff
@@ -237,18 +242,16 @@
 		. = TRUE //no afterattack
 		if(machine_stat & (NOPOWER|BROKEN))
 			return
-		if(beaker)
-			to_chat(user, "<span class='warning'>A container is already loaded into [src]!</span>")
-			return
+		var/obj/item/reagent_containers/beaker_in = I
```

Comment:
This typecast isnt necessary it looks like, you're not doing anything with this value that requires it.